<launch>
  <!-- robot_diagnostics -->
  <rosparam command="load" file="$(find robot_diagnostics)/config/rosbag_recorder.yaml"/>
  <node pkg="robot_diagnostics" type="rosbag_recorder" respawn="false" name="rosbag_recorder" output="screen">
  </node>
</launch>
